<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNewScientist</class>
 <widget class="QDialog" name="addNewScientist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new Computer Scientist to Database</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="2">
    <widget class="QLineEdit" name="lineEdit_link">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>This is a text input, you can use it to enter the website link of the computer scientist.</string>
     </property>
     <property name="placeholderText">
      <string>http://www...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEdit_name">
     <property name="toolTip">
      <string>sdfdsr</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>This is a text input, you can use it to enter the name of the computer scientist.</string>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="accessibleDescription">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Enter letters, no numbers allowed.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelEntGender">
     <property name="text">
      <string>Gender:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelEntName">
     <property name="text">
      <string>Enter Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBox_gender">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>This is a selection input, you can use it to enter the gender of the computer scientist.</string>
     </property>
     <item>
      <property name="text">
       <string>Female</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Male</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unspecified</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelEntDeath">
     <property name="text">
      <string>Year of Death:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelEntBirth">
     <property name="text">
      <string>Year of Birth:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelEntDescr">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QTextEdit" name="textEdit_descr">
     <property name="toolTip">
      <string>SSSS</string>
     </property>
     <property name="whatsThis">
      <string>This is a text input, you can use it to enter the description of the computer scientist.</string>
     </property>
     <property name="placeholderText">
      <string>Enter a detailed description of the life and career the scientist.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelEntLink">
     <property name="text">
      <string>Website Link:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEdit_yod">
     <property name="placeholderText">
      <string>Leave blank if Computer Scientist is still alive.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit_yob">
     <property name="placeholderText">
      <string>Enter year of birth, ex. 1984 or 69</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QDialogButtonBox" name="buttonBox_editScientist">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="labelSciid">
     <property name="text">
      <string>SciID</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addNewScientist</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addNewScientist</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
